### PR TITLE
3 create readme

### LINKED ISSUE #3: Create README setup file
Update README.md on how to setup the site for self-hosting

### PR DESCRIPTION
**Pull request by Dookss**
**Fixes #3** 
First review, more so of a test.
Still need to format and figure out the language a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+# Resume
+Website for generating beautiful resumes that are print friendly
+&nbsp;
+
+## Get started
+#
+### Prerequsites
+* Node
+* MySQL
+&nbsp;
```
Comment (szaveri) [inline README.md:10]: Remove this

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+
+Create a database
+
+
+Create .env in the root folder. &nbsp;
```
Comment (szaveri) [inline README.md:15]: Remove ending space &nbsp;

File: README.md [hunk 3]
```diff
@@ -1 +1,36 @@
-Website for generating beautiful resumes that are print friendly
\ No newline at end of file
+# Resume
+Website for generating beautiful resumes that are print friendly
+&nbsp;
```
Comment (szaveri) [inline README.md:3]: Remove

File: README.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+
+## Get started
+#
```
Comment (szaveri) [inline README.md:6]: Remove

File: README.md [hunk 5]
```diff
@@ -1 +1,36 @@
...
+* Node
+* MySQL
+&nbsp;
+
+Create a database
+
+
+Create .env in the root folder. &nbsp;
+
+Add following:
```
Comment (szaveri) [inline README.md:17]: Add "the" following

File: README.md [hunk 6]
```diff
@@ -1 +1,36 @@
...
+
+```javascript
+NEXTAUTH_URL=http://localhost:3000
+NEXTAUTH_SECRET=*
```
Comment (szaveri) [inline README.md:21]: Change `*` to `<your secret>`

File: README.md [hunk 7]
```diff
@@ -1 +1,36 @@
...
+DATABASE_URL=mysql://root:@127.0.0.1:3306/ Databasename
```
Comment (szaveri) [inline README.md:22]: `DATABASE_URL=mysql://<user>:<password>@<HOST>:<PORT>/<database name>`

File: README.md [hunk 8]
```diff
@@ -1 +1,36 @@
...
+```
+Start MySQL
+
+Start the prisma migration
```
Comment (szaveri) [inline README.md:26]: Run Prisma migration

**Review by szaveri — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Dookss) [reply README.md:5]: No.
This looks better because of the line.

**Review by szaveri — APPROVED** [review]
LGTM - good start

**Pull request merged**